config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   resources :status, only: [:index]
+  resources :proceeding_types
```

Comment:
This generates all actions for proceeding types, if we're only expected to list them this should be restricted as such:

```ruby
resources :proceeding_types, only: [:index]
```